### PR TITLE
feat(#334): Update Dashboard UI

### LINKED ISSUE #334: Update dashboard UI
Currently, the site has the default dashboard that comes with laravel. We need to update it to match the client's page standards
### What needs to be done
 
Update the dashboard vue so that it either uses one we create or update the default one made by laravel/jetstream.
If changes are drastic, make sure tests are update to match.

### PR DESCRIPTION
**Pull request by Alex-Hayeur**
Updated the Dashboard UI for admins, showing them app statistics depending on their roles.

##### _What existing problem does the pull request solve?_
Adds more information on an admin's dashboard, depending on what permissions they have (users who have "users" permissions can see user stats, rooms permissions show room stats etc.

##### _Demonstrate the code is solid. </br> Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI._
Admin Dashboard
Dash1

Regular User Dashboard
Dash2

##### _List all issues this PR resolves_
- closes #334 

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### Sub-tasks Checklist
- [x] UI Implementation
- [x] UI Test
- [x] Backend Implementation
- [x] Integration Tests

### CODE REVIEW

File: resources/js/Components/DashboardMetrics.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    Get more components here 👉 https://web3templates.com/components
+
+    Copyright © 2021
+    =======================================================================
+ -->
+<template>
+	<div class="container max-w-6xl px-5 mx-auto">
+		<div class="grid gap-7 sm:grid-cols-2 lg:grid-cols-4">
+			<div class="p-5 bg-white rounded shadow-sm" v-for="(value, name) in metrics" :key="name">
+				<div class="text-base text-gray-400 ">{{name}}</div>
```
Comment (a-drew) [inline resources/js/Components/DashboardMetrics.vue:19]: could be a good idea to add the tailwind capitalize class here. basically a text decoration to force the first letter of each word to be capitalized. https://tailwindcss.com/docs/text-transform#capitalize

**Review by a-drew — DISMISSED** [review]
small improvement, but otherwise great job with the dashboard

File: resources/js/Components/DashboardMetrics.vue (hunk 1 above)
Comment (Alex-Hayeur) [reply resources/js/Components/DashboardMetrics.vue]: Updated to have capitalization

**Review by EvanDime — APPROVED** [review]
Looks cleaner for admins. Good job!

**Review by a-drew — APPROVED** [review]

**Pull request closed without merging**